src/subprocess-posix.cc

```diff
@@ -55,8 +55,6 @@ bool Subprocess::Start(SubprocessSet* set, const string& command) {
     Fatal("fork: %s", strerror(errno));
 
   if (pid_ == 0) {
-    close(output_pipe[0]);
```

Comment:
Why is this needed? This is the parent's side of the pipe so we ought to close it, right?